[2021-03-03 14:23:39,654] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: titanic_pivot.download_titanic_dataset 2021-03-03T11:23:38.590752+00:00 [queued]>
[2021-03-03 14:23:39,674] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: titanic_pivot.download_titanic_dataset 2021-03-03T11:23:38.590752+00:00 [queued]>
[2021-03-03 14:23:39,674] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-03 14:23:39,674] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-03-03 14:23:39,674] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-03 14:23:39,681] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): download_titanic_dataset> on 2021-03-03T11:23:38.590752+00:00
[2021-03-03 14:23:39,684] {standard_task_runner.py:52} INFO - Started process 49715 to run task
[2021-03-03 14:23:39,686] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'titanic_pivot', 'download_titanic_dataset', '2021-03-03T11:23:38.590752+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/titanicTest.py', '--cfg-path', '/tmp/tmpdtel2_rh', '--error-file', '/tmp/tmpzeyiyi64']
[2021-03-03 14:23:39,687] {standard_task_runner.py:77} INFO - Job 4: Subtask download_titanic_dataset
[2021-03-03 14:23:39,722] {logging_mixin.py:104} INFO - Running <TaskInstance: titanic_pivot.download_titanic_dataset 2021-03-03T11:23:38.590752+00:00 [running]> on host oleg-desktop
[2021-03-03 14:23:39,781] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=download_titanic_dataset
AIRFLOW_CTX_EXECUTION_DATE=2021-03-03T11:23:38.590752+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-03T11:23:38.590752+00:00
[2021-03-03 14:23:42,169] {python.py:118} INFO - Done. Returned value was: None
[2021-03-03 14:23:42,206] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=download_titanic_dataset, execution_date=20210303T112338, start_date=20210303T112339, end_date=20210303T112342
[2021-03-03 14:23:42,264] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-03 14:23:42,301] {local_task_job.py:146} INFO - Task exited with return code 0
